src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -65,6 +65,142 @@ namespace conduit { namespace blueprint { namespace mesh {
 } } }
 
 //-----------------------------------------------------------------------------
+// -- begin internal helper functions --
+//-----------------------------------------------------------------------------
+
+//-----------------------------------------------------------------------------
```

Comment:
@cyrush: When you get a chance, could you take a look at these functions below (i.e. `verify_*_field`) and let me know whether or not they're at a good level of abstraction?  I just want to verify that these functions are close to the abstraction level that we discussed so that I can make changes if necessary before the PR is merged.